modules/schema/src/main/resources/lucuma/odb/graphql/OdbSchema.graphql

```diff
@@ -4762,34 +4762,34 @@ type Query {
   ): ProgramSelectResult!
 
   # Observations grouped by commonly held science mode
-  observingModeGroup(
-    # Program ID
-    programId: ProgramId!
+  # observingModeGroup(
+  #   # Program ID
+  #   programId: ProgramId!
 
-    # Filters the selection of observations.
-    WHERE: WhereObservation
+  #   # Filters the selection of observations.
+  #   WHERE: WhereObservation
 
-    # Limits the result to at most this number of matches (but never more than 1000).
-    LIMIT: NonNegInt
+  #   # Limits the result to at most this number of matches (but never more than 1000).
+  #   LIMIT: NonNegInt
 
-    # Set to true to include deleted values
-    includeDeleted: Boolean! = false
-  ): ObservingModeGroupSelectResult!
+  #   # Set to true to include deleted values
+  #   includeDeleted: Boolean! = false
+  # ): ObservingModeGroupSelectResult!
```

Comment:
This will be needed if we're going to have a "Configurations" tab.  This isn't needed for XT?

Reply:
I think it was determined that the configurations tab isn't happening for XT. At least for the moment.